Vagrantfile

```diff
@@ -0,0 +1,35 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+# Author: David Manouchehri
+
+Vagrant.configure("2") do |config|
+	config.vm.box = "bento/ubuntu-16.04"
+
+	config.vm.provision "shell", inline: <<-SHELL
+		# apt-get update
+		# DEBIAN_FRONTEND=noninteractive apt-get -y upgrade
+		snap install --classic go
+		# apt-get clean
```

Comment:
I guess these comments are obsolete and can be removed?

Reply:
They're good to leave in; they're optional in case someone needs/wants to update all their packages. (This adds a couple minutes onto the build though.)

Reply:
They look like cruft, if they aren't please leave a descriptive comment.

Reply:
I don't think basic `apt-get` commands needs to explained, but I've removed them anyway.